util/devel/chplspell-dictionary

```diff
@@ -2215,7 +2215,7 @@ oleg
 opendir
 openfd
 openfp
-openmem
+openMemFile
```

Comment:
I'd just add the line to this file, rather than deleting the `openmem` entries, otherwise a spell-check run with still complain about the deprecated function name.